SEIR/test/data/filter.txt

```diff
@@ -0,0 +1,32 @@
+0 0
```

Comment:
Looking at this, let's find some way to integrate with COVID19_Minimal (or future example repository) rather than maintaining a filter here.